src/Microsoft.ML.DataView/DataViewType.cs

```diff
@@ -21,7 +21,7 @@ public abstract class DataViewType : IEquatable<DataViewType>
         /// <summary>
         /// Constructor for extension types, which must be either <see cref="PrimitiveDataViewType"/> or <see cref="StructuredDataViewType"/>.
         /// </summary>
-        private protected DataViewType(Type rawType)
+        protected DataViewType(Type rawType)
```

Comment:
This is intentional. It shouldn't be changed. See the summary comment.

Reply:
Ok. Will derive custom types from `StructuredDataViewType`.

---
In reply to: [285623815](https://github.com/dotnet/machinelearning/pull/3263#discussion_r285623815) [](ancestors = 285623815)